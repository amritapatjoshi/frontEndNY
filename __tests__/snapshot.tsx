import { render } from '@testing-library/react'
import UseEnv from '@/app/examples/use-env/page'

it('renders UseEnv page unchanged', () => {
  const { container } = render(<UseEnv />)
  expect(container).toMatchSnapshot()
})
