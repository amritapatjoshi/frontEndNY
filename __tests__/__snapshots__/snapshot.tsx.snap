// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders UseEnv page unchanged 1`] = `
<div>
  <div
    class="wrapper"
  >
    <h1>
      Value from environment variable: 
      https://test-local.com
    </h1>
  </div>
</div>
`;
